### ISSUE TITLE
DBeaver client not opening due to "Jaava heap space" error. Heap has already been manually upped to 4GB.

### ISSUE THREAD

**Issue by GreenGem-ButOnGitHub** [dbeaver/dbeaver#12510]
#### System information: 
- Windows 10 Home 64-bit (10.0, Build 19041)
- DBeaver 21.0.5 x86

#### Connection specification:
- This is an issue with the dbeaver client, not a specific database.

#### Describe the problem you're observing:
Running "dbeaver.exe" causes the launcher to appear (see image below), hang on "Starting DBeaver," then disappear. This creates a dbeaver background process, but does NOT open the client.
![image](https://user-images.githubusercontent.com/11450137/118563619-e1d9cc00-b73c-11eb-8f87-e45e45ac63ac.png)
According to the attached log file (see bottom of post), this is due to a java.lang.OutOfMemory error "UTF16 String size is 1073872894, should be less than 1073741823."

Running "dbeaver-cli.exe" instead causes this:
[Console.txt](https://github.com/dbeaver/dbeaver/files/6497180/Console.txt)
The notable line is "Internal error in workbench lifecycle java.lang.OutOfMemoryError: Java heap space"

What's odd is that I've already modified the "dbeaver.ini" file to read "-Xmx4096m" and the Java runtime parameters to contain the same.

#### Steps to reproduce, if exist:
Run dbeaver.exe or dbeaver-cli.exe; that's it.

#### Include any warning/errors/backtraces from the logs
From running dbeaver.exe:
`java.lang.OutOfMemoryError: UTF16 String size is 1073872894, should be less than 1073741823`

From running dbeaver-cli.exe:
`Internal error in workbench lifecycle java.lang.OutOfMemoryError: Java heap space`
[dbeaver-debug.log](https://github.com/dbeaver/dbeaver/files/6497196/dbeaver-debug.log)

**Comment by kseniiaguzeeva** [comment]
Please try to [reset workspace](https://github.com/dbeaver/dbeaver/wiki/Reset-workspace). It should help.

**Comment by uslss** [comment]
there is no update on the issue for a long time.
If it is still actual for you - feel free to reopen it with additional info

**Comment by davidh-cloudster** [comment]
Hi 
I am having the same issues 
OS: Windows 10 Home
DBeaver version: 22.2.0

In the dbeaver.ini , I have the following setting:

-vmargs
-XX:+IgnoreUnrecognizedVMOptions
--add-modules=ALL-SYSTEM
-Dosgi.requiredJavaVersion=11
-Xms64m
-Xmx4096m
-Djavax.net.ssl.trustStoreType=WINDOWS-ROOT
-Ddbeaver.distribution.type=exe

I only changed the old value to -Xmx4096m.
I had Java heap space Error before I change the old value
However, after I changed the value to -Xmx4096m
I got a new error:
 UTF16 String size is 2098200574, should be less than 1073741823
 
I tried to reset workspace and use dbeaver.exe or dbeaver-cli.exe to open DBeaver
Both way give me same error
What should I do?
The SQL file that I am trying to run is 16.4 GB
Is it the cause of the error?
